SU2_CFD/src/numerics/flow/flow_sources.cpp

```diff
@@ -313,22 +313,32 @@ CNumerics::ResidualType<> CSourceIncAxisymmetric_Flow::ComputeResidual(const CCo
       Eddy_Viscosity_i       = V_i[nDim+5];
       Thermal_Conductivity_i = V_i[nDim+6];
 
-      su2double total_viscosity;
+      // nijso: FIXME: the viscous terms have serious convergence issues...
+
 
       total_viscosity = (Laminar_Viscosity_i + Eddy_Viscosity_i);
+      // su2double total_conductivity = Thermal_Conductivity_i + Cp_i*Eddy_Viscosity_i/Prandtl_Turb;
 
       /*--- The full stress tensor is needed for variable density ---*/
       ComputeStressTensor(nDim, tau, PrimVar_Grad_i+1, total_viscosity);
 
       /*--- Viscous terms. ---*/
 
       residual[0] -= 0.0;
-      residual[1] -= Volume*(yinv*tau[0][1] - TWO3*AuxVar_Grad_i[0][0]);
-      residual[2] -= Volume*(yinv*2.0*total_viscosity*PrimVar_Grad_i[2][1] -
-                             yinv* yinv*2.0*total_viscosity*Velocity_i[1] -
-                             TWO3*AuxVar_Grad_i[0][1]);
+      //residual[1] -= Volume*(yinv*tau[0][1] - TWO3*AuxVar_Grad_i[0][0]);
+      //residual[2] -= Volume*(yinv*2.0*total_viscosity*PrimVar_Grad_i[2][1] -
+      //                       yinv* yinv*2.0*total_viscosity*Velocity_i[1] -
```

Comment:
I dont understand it either? @mheimgartner 
(maybe removing the viscous contribution?)

Reply:
This was my doing, I did some test where I removed the terms related to  the divergence of velocity, since they should be 0 for low mach cases. I did not find any big changes in convergence. I didn't do a lot of tests, but I think it is safe to simply keep the nabla.V terms.